src/pages/MainPage.vue

```diff
@@ -327,11 +326,22 @@ export default {
       deep: true
     }
   },
+  created () {
+    if (parseInt(this.$cookies.get('versionNumber')) !== this.$store.state.versionNumber) {
+      console.log('Warning: the version number has changed.');
+      // do whatever needs to happen when the version changed, probably including clearing local storage
+      // then update the version number cookie
+      localStorage.clear();
```

Comment:
I appreciate you doing this part on a separate branch and then adding this line here, that's exactly the best practice (IMO)